IFC 4.3/Superelevation_1/UT_Superelevation_1.IFC

```diff
@@ -0,0 +1,484 @@
+#57=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#58);
+#58=IFCCIRCULARARCSEGMENT2D(#59,5.702586430944,79.33785465,50.,.T.);
+#59=IFCCARTESIANPOINT((335227.52147844,4084557.67048984));
+#60=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#61);
+#61=IFCTRANSITIONCURVESEGMENT2D(#62,1.00615821607531,32.,50.,0.,.T.,.F.,.CLOTHOIDCURVE.);
+#62=IFCCARTESIANPOINT((335297.1868326,4084572.7216982));
+#63=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#64);
+#64=IFCLINESEGMENT2D(#65,1.32615821602658,12.39520568);
+#65=IFCCARTESIANPOINT((335308.14596689,4084602.63177967));
+#66=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#67);
+#67=IFCTRANSITIONCURVESEGMENT2D(#68,1.32615821602658,41.66666667,0.,60.,.F.,.F.,.CLOTHOIDCURVE.);
+#68=IFCCARTESIANPOINT((335311.14815049,4084614.6579186));
+#69=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#70);
+#70=IFCCIRCULARARCSEGMENT2D(#71,0.978935993872235,27.60658519,60.,.F.);
+#71=IFCCARTESIANPOINT((335325.75784196,4084653.44126307));
+#72=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#73);
+#73=IFCTRANSITIONCURVESEGMENT2D(#74,0.518826240527787,41.66666667,60.,0.,.F.,.F.,.CLOTHOIDCURVE.);
+#74=IFCCARTESIANPOINT((335345.80042437,4084672.07101789));
+#75=IFCALIGNMENT2DHORIZONTALSEGMENT(.F.,$,$,#76);
+#76=IFCLINESEGMENT2D(#77,0.171604018373445,35.3941232);
+#77=IFCCARTESIANPOINT((335385.54643726,4084683.81177391));
+#78=IFCRELCONTAINEDINSPATIALSTRUCTURE('12Qm7Jv3zDahoI$_VaurZ7',$,$,$,(#30),#26);
+#79=IFCALIGNMENT2DVERSEGLINE($,$,$,0.,14.256,365.8,0.0784810126582277);
+#80=IFCALIGNMENT2DVERSEGPARABOLICARC($,$,$,14.256,129.488,366.918825316456,0.0784810126582277,890.,.T.);
+#81=IFCALIGNMENT2DVERSEGLINE($,$,$,143.744,299.295,367.661484536082,-0.0670103092783505);
+#82=IFCALIGNMENT2DVERSEGPARABOLICARC($,$,$,443.039,47.922,347.605634020619,-0.0670103092783505,260.,.F.);
+#83=IFCALIGNMENT2DVERSEGLINE($,$,$,490.961,16.106,348.810709561484,0.117303516609678);
+#84=IFCALIGNMENT2DVERTICAL((#79,#80,#81,#82,#83));
+
+/* Superelevation is related to this element */
+#85=IFCFACILITYPART('2nEsjUsrn8pfQJ8laIXY5h',$,$,$,'SEGMENT',$,$,$,.COMPLEX.,$,.LONGITUDINAL.);
+
+/* Inclusion of FacilityPart in Road */
+#86=IFCRELAGGREGATES('2MOXKugMH2SgeNpvGA2na0',#5,$,$,#28,(#85));
+
+/* 
+  Repeated in every IFCSUPERELEVATIONEVENT 
+ */
+
+/* 3D position */
+#87=IFCDIRECTION((0.999264319742782,-0.0383512618957454,0.));
+#88=IFCDIRECTION((0.,0.,1.));
+#89=IFCCARTESIANPOINT((335085.95782221,4084594.13214501,365.8));
+
+/* Three Properties*/
+#91=IFCPROPERTYSINGLEVALUE('Superelevation',$,IFCRATIOMEASURE(-0.02),$);
+#93=IFCPROPERTYSINGLEVALUE('Side',$,IFCTEXT('BOTH'),$);
+#95=IFCPROPERTYSINGLEVALUE('Transition',$,IFCTEXT('CONST'),$);
+
+/* Linear placement */
+#96=IFCDISTANCEEXPRESSION(0.,$,$,$,$);
+#97=IFCORIENTATIONEXPRESSION(#87,#88);
+#98=IFCAXIS2PLACEMENT3D(#89,#88,#87);
+#99=IFCLINEARPLACEMENT($,#31,#96,#97,#98);
+
+/* IfcEventElement of Superelevation type */
+#100=IFCEVENTELEMENT('3IgH4skvr0$eWM9o1FvG6j',$,$,$,$,#99,$,.SUPERELEVATION.);
```

Comment:
As noted by @jmirtsch , this is not yet up-to-date with the recent schema.

Reply:
Updated.